libs/editor-common/assets/ZSSRichTextEditor.js

```diff
@@ -2606,8 +2606,24 @@ ZSSEditor.completeListEditing = function() {
             if (node.nodeType == 1 &&
                     (node.tagName.toUpperCase() == NodeName.UL
                         || node.tagName.toUpperCase() == NodeName.OL)) {
-                // Make a new P node as a sibling to the parent node
-                document.execCommand('insertParagraph', false);
+
+                var focusedNode = document.getSelection().getRangeAt(0).startContainer;
+
+                if (focusedNode.nodeType == 3) {
```

Comment:
Note: if we have to clean ZSSEditor someday, it would be great to use const static var for node types, even if we know what the 3 is for, it's always better for reading the code ;).